react-common/components/theming/ThemeCard.tsx

```diff
@@ -13,8 +13,10 @@ export const ThemeCard = (props: ThemeCardProps) => {
         <Card
             className="theme-card"
             role="listitem"
+            aria-label={theme.name}
             key={theme.id}
             onClick={() => onClick(theme)}
+            tabIndex={onClick && 0}
```

Comment:
Should `onClick` just be necessary?

Reply:
It could be, but I don't think the check hurts anything. I'm following a pattern that we already have in the extension cards.